src/app/pages/storage/modules/disks/components/disk-form/disk-form.component.ts

```diff
@@ -64,7 +63,11 @@ export class DiskFormComponent implements OnInit {
 
   setFormDisk(disk: Disk): void {
```

Comment:
Instead of adding a null check here, we can avoid it by not passing null when calling this method in the disk-list component for example. Let's find other places where this method is used and make sure that null isn't passed for this values.

Reply:
I think it’s better to keep my fix, because other places just call this method, so It’s better to fix it in single place rather that in all places like: 
editForm.setFormDisk(this.disk)

Reply:
@RehanY147